[Zscaler Internet Access] Fixed an issue where the zscaler-logout and zscaler-activate-changes returned an error (#36867)
demisto/content: Packs/Zscaler/ReleaseNotes/1_3_27.md

#### Integrations

##### Zscaler Internet Access

> Fixed an issue where the **zscaler-logout** and **zscaler-activate-changes** commands did not work properly and returned an error.

Reviewer: ```suggestion
Fixed an issue where the **zscaler-logout** and **zscaler-activate-changes** commands returned an error after a successful execution.
```
